http/README.md

```diff
@@ -74,7 +73,7 @@ This object contains information about the message representation in HTTP.
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="messageBindingObjectHeaders"></a>`headers` | [Schema Object][schemaObject] | A Schema object containing the definitions for HTTP-specific headers. This schema MUST be of type `object` and have a `properties` key.
+<a name="messageBindingObjectHeaders"></a>`headers` | [Parameters Object][parametersObject] | A Parameters object containing the definitions for HTTP-specific headers.
```

Comment:
I'm a bit concerned that we're using the Parameters Object for header definitions. I think I understand the change but not sure how this would evolve over time 🤔

Reply:
same as with `websocket`, this should not change imho